framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -199,6 +199,12 @@
     // org.checkerframework.framework.source.SourceeVisitor.checkForSuppressWarningsAnno
     "warnUnneededSuppressions",
 
+    // Require that warning suppression annotations contain a checker key as a prefix in order for
+    // the warning to be suppressed.
+    // org.checkerframework.framework.source.SourceChecker.checkSuppressWarnings(java.lang.String[],
+    // java.lang.String)
+    "requireCheckerPrefixInWarningSuppressions",
```

Comment:
This name is pretty bad. Maybe just `requireCheckerInWarningSuppressions`?  Other ideas?

Reply:
I think a long name is OK for this rarely-used feature.
I changed it to `requirePrefixInWarningSuppressions`.